testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2351,6 +2352,39 @@ public void testRenameSchema()
         }
     }
 
+    @Test
+    public void testDropSchemaCascade()
+    {
+        if (!hasBehavior(SUPPORTS_DROP_SCHEMA_CASCADE)) {
+            String schemaName = getSession().getSchema().orElseThrow();
+            assertQueryFails(
+                    "DROP SCHEMA " + schemaName + " CASCADE",
+                    "This connector does not support dropping schemas with CASCADE option");
+            return;
```

Comment:
If i implement DROP SCHEMA CASCADE support for BigQuery, and forget to update this test, will it drop the shared testing resource's schema?

let's maybe try drop-cascade some new schema (easy if you also move `skipTestUnless(hasBehavior(SUPPORTS_CREATE_SCHEMA));` up to be first)